pkgs/tools/system/swaysettings/default.nix

```diff
@@ -0,0 +1,75 @@
+{
+  pkgs,
+  meson,
+  vala,
+  pkg-config,
+  gettext,
+  ninja,
+  python3Full,
+  wrapGAppsHook,
+  appstream-glib,
+  desktop-file-utils,
+  gsettings-desktop-schemas,
+  glib,
+  gtk3,
+  libhandy,
+  json-glib,
+  libgee,
+  pantheon,
+  libxml2,
+  accountsservice,
+  gobject-introspection,
+  dbus
+}:
+with pkgs;
+pkgs.stdenv.mkDerivation rec {
+  pname = "swaysettings";
```

Comment:
Are you using Notepad or whatever? Indent the code.

Reply:
No. My nixfile formater was not activated correctly. Corrected